test/integration/pr_test.go

```diff
@@ -61,7 +67,7 @@ func TestPRGithubDecorationFailure(t *testing.T) {
 		os.Getenv(prGithubRepoName),
 	}
 	err, _ := executeCommand(t, args...)
-	assert.ErrorContains(t, err, "Failed creating github PR Decoration")
+	assert.ErrorContains(t, err, "scan not found")
```

Comment:
this is ther err-
"A PR couldn't be created for this scan because it is still in progress."
why it returns something else?